### DISCUSSION TITLE
Changing the colors on the RGB strip

### DISCUSSION THREAD

**Discussion by EX8F22** [onlaj/Piano-LED-Visualizer#245] [General]
Hello,

First of all thank you for putting this out for all of us to enjoy, it has been amazing.

I would like to know if there's a way that I could change each individual RGB light to a different color? I have a pattern I'd like to set for each key/octave. 

Is this possible?

**Comment by onlaj** [comment]
Hi.
You can use multicolor to set any key to any color you like.
